WebContent/js/components/JobFile.js

```diff
@@ -38,6 +39,7 @@ class JobFile extends React.Component {
         } else {
             dispatch(fetchJobFile(job.get('jobName'), job.get('jobId'), file.label, file.id));
         }
+        dispatch(updateTitle(`${job.get('jobId')}-${file.label}`));
```

Comment:
I don't think this is the best place to change the page title, we have multiple ways a user can open content such as single clicking on a file like you've seen here, but also a user can right click on a job instance and select Open to view the concatenated view of all files or they can select Get JCL(SJ) which will get the JCL used to submit a given job. All 3 of these actions should trigger a page title update

I think the best place to dispatch the action to update the page title would be the dispatchReceiveContent function in actions/content.js

I would also like to always prefix the job file with JES Explorer

Reply:
I have had a look at the dispatchReceiveContent function and currently the getJCL function is not making a call to it, if you can update the getJCL function to also dispatch it's receive content via the dispatchReceiveContent function that would be great!